src/connector.js

```diff
@@ -983,9 +983,9 @@ function updateResultListState( newState ) {
 			}
 
 			let breadcrumb = "";
-			let printableUri = stripHtml( result.printableUri );
-			let clickUri = stripHtml( result.clickUri );
-			let title = stripHtml( result.title );
+			let printableUri = DOMPurify.sanitize( result.printableUri );
+			let clickUri = DOMPurify.sanitize( result.clickUri );
+			let title = DOMPurify.sanitize( result.title );
```

Comment:
@bblaisATcoveo Please bring this one back to `stripHTML` since we are not expecting any HTML (as output) as part of this value and therefore want to remove it.